src/Tests/dotnet.Tests/.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.cs]
+
+# CS0618: Type or member is obsolete
+dotnet_diagnostic.CS0618.severity = suggestion
```

Comment:
I'm adding this config in a few places because System.CommandLine has recently marked a few of their APIs as obsolete. We use them across quite a few places in the CLI and after discussing with @jonsequitur there doesn't seem to be a simple fix for this. I'm tracking making a better fix here: https://github.com/dotnet/sdk/issues/18952 but I don't think this work is feasible for .NET 6 timeframe.